src/_ert/forward_model_runner/reporting/event.py

```diff
@@ -87,7 +87,9 @@ def __init__(
             # can be finished but not all the events were sent yet
             self._finished_event_timeout = 600
 
-    def stop(self):
+    def stop(self, exited_event: Exited | None = None):
+        if exited_event:
+            self._statemachine.transition(exited_event)
```

Comment:
Reading the source code of `statemachine.py` I would think you would need to add the Exited state to `_transitions` for the transition to be accepted?

Reply:
me blind 🙈